### DISCUSSION TITLE
▸ Generating 'App.app.dSYM': Failed to Archive iOS Ionic App

### DISCUSSION THREAD

**Discussion by sebimarkgraf** [codemagic-ci-cd/codemagic-docs#1610] [Q&A] [answered]
Hey,

I am stuck with getting my Ionic app to build for iOS. 
While the android app works and the ios build runs locally,
the codemagic version always fails at generating the App.app.dysm.

I did not find similar issues in here or elsewhere. And already tried the debugging the step with SSH but with no success.

I failed build can be found as ID: 6450b4b248d4a608726e5d6a

I have attached the logs of the corresponding build step.

[build_ipa_for_distribution(1).log](https://github.com/codemagic-ci-cd/codemagic-docs/files/11370635/build_ipa_for_distribution.1.log)

Thank you very much for any help!

**Comment by icarusdust** [comment, marked as answer]
Hi @sebimarkgraf, I am reading the following error message in the logs:

`rsync error: some files/attrs were not transferred at main.c`

It is an issue where Xcode 14.3 wants you to update your Cocoapods version from 1.12.0. So, just set `cocoapods` from `default` in `codemagic.yaml` to `1.12.1`

**Reply by sebimarkgraf** [reply]
Thank you very much @icarusdust !
Works like a charm now.

Is there a place to lookup such common errors and the underlying problem?
The CodeMagic FAQ on ionic apps did not include this.

**Reply by icarusdust** [reply]
Glad to hear that it worked. It is not related to ionic or any other platform @sebimarkgraf but Cocoapods itself. There is a very long discussion [here](https://github.com/CocoaPods/CocoaPods/issues/11808), you can check it. 
I'll see where we can mention this in our common issues docs page.
